### ISSUE TITLE
Expo SDK 50 compatibility

### ISSUE THREAD

**Issue by 7nohe** [EvanBacon/expo-apple-targets#25]
Thanks for creating this plugin!
It would be nice if this plugins is supporting Expo SDK 50.

I attempted to use this plugin and encountered the following error while executing the `npx expo prebuild` command.

```
✔ Finished prebuild
⚠️  Something went wrong running `pod install` in the `ios` directory.
Command `pod install` failed.
└─ Cause: An error occurred while processing the post-install hook of the Podfile.

[!] The plist file at path `/Users/me/mypath/myproject/ios/Info.plist` doesn't exist.
```

**Comment by 7nohe** [comment]
I found a similar issue, and it seems that the issue may be solved by upgrading React Native from 0.73.2 to 0.73.3.
https://github.com/OneSignal/onesignal-expo-plugin/issues/213#issuecomment-1914788799

**Comment by 7nohe** [comment]
> I found a similar issue, and it seems that the issue may be solved by upgrading React Native from 0.73.2 to 0.73.3. [OneSignal/onesignal-expo-plugin#213 (comment)](https://github.com/OneSignal/onesignal-expo-plugin/issues/213#issuecomment-1914788799)

It worked with this. Expo was not directly involved.
